src/cldr.js

```diff
@@ -4,7 +4,7 @@
 var expDTComponents = /(?:[Eec]{1,6}|G{1,5}|(?:[yYu]+|U{1,5})|[ML]{1,5}|d{1,2}|a|[hkHK]{1,2}|m{1,2}|s{1,2}|z{1,4})(?=([^']*'[^']*')*[^']*$)/g;
 
 // Skip over patterns with these datetime components
-var unwantedDTCs = /[QxXVOvZASjgFDwWIQqH]/;
```

Comment:
enabling `H` as a way to produce 24H `hour` values. I'm not sure why was it there in the first place. /cc @andyearnshaw